katello.gemspec

```diff
@@ -44,11 +44,11 @@ Gem::Specification.new do |gem|
   gem.add_dependency "anemone"
 
   #pulp3
-  gem.add_dependency "pulpcore_client", "<= 3.0.0rc8.dev01573125401"
-  gem.add_dependency "pulp_file_client", "<= 0.1.0b5.dev01573069180"
-  gem.add_dependency "pulp_ansible_client", "<= 0.2.0b6.dev01573147224"
-  gem.add_dependency "pulp_docker_client", "<= 4.0.0b8.dev01573136246"
-  gem.add_dependency "pulp_rpm_client", "<= 3.0.0b8.dev01573228480"
+  gem.add_dependency "pulpcore_client", "<= 3.0.0rc8"
+  gem.add_dependency "pulp_file_client", "<= 0.1.0b5.dev01573759926"
```

Comment:
should this be https://rubygems.org/gems/pulp_file_client/versions/0.1.0rc1  ?

Reply:
yep!